utils/e2e-tests/ts/run-tests

```diff
@@ -10,6 +10,6 @@ shift
 HUMANODE_PEER_PATH="$(realpath -s "$HUMANODE_PEER_PATH")"
 export HUMANODE_PEER_PATH
 
-if [[ "$TEST_SUITE" == "base" ]]; then
-  exec yarn utils/e2e-tests/ts test
+if [[ -d "$SCRIPT_DIR/tests/$TEST_SUITE"/ ]]; then
```

Comment:
Not sure we need this check; doesn't it make sense to fail if a non-existent test suite is requested?

Reply:
For the maintenance purposes - I'd say this definitely makes sense... Once can easily miss some place to be updated upon renames - we'd want this to be very visible

Reply:
Is this bad? Does it actually block anything? What if allow tests to fail if there are no tests to run?

Reply:
I suppose we want to avoid spawning CI jobs when they do no useful work...

Reply:
> Is this bad? Does it actually block anything? What if allow tests to fail if there are no tests to run?

The whole CI job is failed as 1 code error is returned

Reply:
Makes sense actually - why would we want a I job **not** to fail if no tests were run? We will just *not run the job* instead :D

Reply:
What do you think? Let's make the test suites / subsystems follow this logic?

Reply:
Yes, kind of!

But better yet, express it more explicitly via the matrix itself:

```yaml
          - suite: "base"
            build-args: ""
          - suite: "try-runtime"
            build-args: "--features try-runtime"
            skip-ts-tests: true
          - suite: "evm-tracing"
            build-args: "--features evm-tracing"
            skip-bash-tests: true
```

```yaml
if: matrix.test.skip-ts-tests != true
```

Reply:
Improved

Thanks for the idea, now it's much better!